spec/requests/users_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'users', type: :request do
+  describe 'GET/index' do
+    before :each do
+      get '/users'
+    end
+    it 'is a success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'should render index' do
+      expect(response).to render_template(:index)
+    end
+    it 'Should return the correct placeholder' do
+      expect(response.body).to include('Users')
+    end
+  end
+
+  describe 'GET/show' do
+    before :each do
+      get '/users'
+    end
+    it 'is a success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'Should return the correct placeholder' do
+      expect(response.body).to include('Users')
+    end
+  end
```

Comment:
- [ ] Please add a test that ensures the correct view template is rendered.
[Hint]: like that:
```ruby
it 'renders the correct template' do
    expect(response).to render_template(:show)
end
```